runtime/lua/vim/lsp/handlers.lua

```diff
@@ -61,7 +61,7 @@ local function progress_handler(_, result, ctx, _)
     client.messages.progress[token].done = true
   end
 
-  vim.api.nvim_command('doautocmd <nomodeline> User LspProgressUpdate')
+  a.nvim_command('doautocmd <nomodeline> User LspProgressUpdate')
```

Comment:
Can be replaced with `nvim_exec_autocmds`.

Reply:
I'd prefer to do one thing at the time honestly, can it wait for another PR?

Reply:
Yup; I thought it was the only obsolete `nvim_command`, but there's a few others like that, so indeed better left to a dedicated PR.